### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by HatimDev-HE**
In this feature, I added a header with the text Hello Microverse and styled it with my favorite color.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Linters
 
 on: pull_request
 
-env:git 
+env:
```
Comment (vechicin) [inline .github/workflows/linters.yml:5]: You shouldn't change any linters configurations as stated in the [linters config](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) documentation. Kindly fix this.

File: Linter1 [hunk 2]
```diff
@@ -0,0 +1 @@
+N/A
```
Comment (vechicin) [inline Linter1:1]: Please remove this file, it doesn't serve any functionality and it makes your project look unprofessional.

File: styles.css [hunk 3]
```diff
@@ -1,3 +1 @@
-h1 {
-    background-color: purple;
-}
\ No newline at end of file
+h1 { background-color: purple; }
```
Comment (vechicin) [inline styles.css:1]: The requirements state that it is the color of the header, not the background color that needs to change. Kindly change the `background-color` rule to `color`.

[OPTIONAL] It is considered bad practice to use color names. I suggest you change this color to its `hex` or `rgb` equivalent.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @HatimDev-HE  ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Linters checks passing
- [x] Correct GitHub Flow
- [x] Correct `.gitignore` use

 ## Required Changes ♻️

- [ ] You need to submit the same PR for every re-review you request. This is the 3rd PR you have submitted for this project. Please stop doing so because for future projects submitting different PR will make your requests invalid and it'll prevent you from moving forward in the curriculum.
- [ ] Please change your PR title and description to something more descriptive. Here's an example:
Title: Hello Microverse
Description: In this feature I added a header with the text `Hello Microverse` and styled it with my favorite color.

- [ ] Your README file needs to follow [this template](https://github.com/microverseinc/readme-template) as part of the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md). Kindly fix this.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (HatimDev-HE) [reply .github/workflows/linters.yml:5]: I fixed it, I had added the git on env: but revised it later on

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @HatimDev-HE, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] As suggested by the previous reviewer kindly add PR Description as `In this feature I added a header with the text Hello Microverse and styled it with my favorite color.` You can add description by clicking [here](https://github.com/HatimDev-HE/Hello-Microverse/pull/3#issue-1248362707) and editing the description by clicking the 3 dots as shown below in the video.

https://user-images.githubusercontent.com/70269507/170440786-439d5d02-f534-430c-a5cb-06684eb08be7.mov

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @HatimDev-HE ,  

Your project is complete! Great job implementing the changes requested by the previous reviewer 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors
- Good readme template

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**